tools/openvino_dev/requirements.txt

```diff
@@ -0,0 +1,9 @@
+addict>=2.4.0
+jstyleson>=0.0.2
+opencv-python
+opencv-python>=4.5
+pandas~=1.3.5
+pillow>=8.1.2
+scipy>=1.8; python_version >= "3.8"
+texttable>=1.6.3
+tqdm>=4.54.1

```

Comment:
why do we need it?

Reply:
In my case, I'm using raspberry pi 2 model b with raspbian11 os. I'm trying to install openvino-dev from github repo because when i do "pip install openvino-dev" it show error like "error module named openvino not found" (i guess there's no pypi support for raspberry pi). So i just go clone this repo and install the openvino-dev directly from "tools/openvino-dev". When i do install, it shows error because the module that i ads to requirements.txt not installed and wont install automatically. So i have to write thar requirements.txt manually and then install it manually too.

Reply:
Please, follow https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build_linux.md#additional-build-options section to build Python API and wheels.
Target for OpenVINO dev wheel is called `openvino_dev_wheel`. Once you have built it, it can be installed from `<build folder>/wheels`

Reply:
It looks that this is your custom requirements file and `openvino-dev` component does not depend on this modules.